tests/test_authorize.py

```diff
@@ -59,7 +60,8 @@ def test_should_fail_if_not_in_list(self):
 class TestAuthorizeWithResource(TestAuthorize):
 
     def _create_fake_resource(self):
-        return FakeResource(set(('admin', 'principle', 'superintendent')))
+        authmap = {'on_get': ['admin', 'principle', 'superintendent']}
```

Comment:
Thats funny because this line did say "Principal" before and then everything failed because it didnt match the actual test - but then I was lazy and decided to make this line wrong instead of all the other ones right. Was hoping I wouldn't get called out.